charts/spire/README.md

```diff
@@ -117,6 +117,7 @@ Now you can interact with the Spire agent socket from your own application. The
 | upstream-spire-agent.telemetry.prometheus.port | int | `9989` |  |
 | upstream.enabled | bool | `false` | enable upstream agent and driver for use with nested spire. |
 | spiffe-csi-driver.agentSocketPath | string | `"/run/spire/agent-sockets/spire-agent.sock"` | The unix socket path to the spire-agent |
+| spiffe-csi-driver.csiDriverLabels | object | `{}` | Labels to apply to the CSIDriver |
```

Comment:
Should this just be `labels` to match the other components?

Reply:
They are specifically on the kind: CSIDriver object by the look of it. Where the other labels tend to be on the workload itself? I think its probably the right thing to do if those labels have to be different then the workload labels.